FranciscoPerez/react-challenge-project-jan-2023
# Fp 3 add time test

## Changes
1. Add test condition to verify order time format from challenge issue #2.

Branched from issue #2 (fp-2-fix-order-time-format)

## Purpose
Current test suite only tests if the component renders one or multiple orders. It does not verify order time format.

[Issue #3](https://github.com/Shift3/react-challenge-project-jan-2023/issues/3)


## Approach
Adds a new test condition to the `ordersList.test.js` file to specifically check order time format.

This new test renders an `OrderList` component using a dummy order whose time contains single-digit minutes and seconds.

## Testing Steps
1. Open a terminal into the "application" folder.
2. Type "npm test" into the terminal window.
3. Type "a" to run all tests.
4. Verify that the `src/components/view-orders/ordersList.test.js` test passed.

## Learning
Reviewed the React Testing Library documentation on their official website.

[React Testing Library](https://testing-library.com/docs/react-testing-library/intro)

[Testing Library Cheat Sheet](https://testing-library.com/docs/react-testing-library/cheatsheet)

Closes #3
